openpype/hosts/maya/api/lib.py

```diff
@@ -3402,13 +3402,22 @@ def get_color_management_preferences():
         )
     }
 
+    # Split view and display from view_transform. view_transform comes in
+    # format of "{view} ({display})".
+    display = data["view_transform"].split("(")[-1].replace(")", "")
+    data.update({
+        "display": display,
+        "view": data["view_transform"].replace("({})".format(display), "")[:-1]
+    })
```

Comment:
Wasn't this splitting only for need for older versions of Maya?

Also, what if a display or view has a `(` in it? I didn't see any for displays or views in Aces 1.0.3 but I did see them in names for Color spaces in that config. I think the logic is a bit to basic and we'd at least need to go regex.

```python
regex =re.compile(r"^(.+) \(.+\)$")
match = regex.match(combined_display_view)
display = match.group(1)
view = match.group(2)
```
Or better:
```python
regex =re.compile(r"^(?P<view>.+) \(?P<display>.+\)$")
match = regex.match(combined_display_view)
display = match.group("view")
view = match.group("display)

# Or:
print(match.groupdict())
# {"view": "My View", "display": "My Display"}
```

This works on these examples:
```
My View (My Display)
My View (Sexy) (My Display)
My View (Hello) (My Display-(Hello))
```
But fails as soon as the display contains a ` (Sub)` branch with a space before the `(`, like:
```
My - hello207 (-view) (my complex display (with brackets with space before it))
```
Which would parse it as:
```
view: My - hello207 (-view) (my complex display
display: (with brackets with space before it))
```

But at least it's an improvement? :)

Reply:
Nice, thanks. Will test and use. My regex skills just werent up to the challenge :)